runtimes/rust/lbr-prelude/src/json.rs

```diff
@@ -0,0 +1,322 @@
+use data_encoding::BASE64;
+use num_bigint::BigInt;
+use serde::ser;
+use serde_json::{self, Error, Value};
+use std::collections::{HashMap, HashSet};
+use std::hash::Hash;
+
+/// Trait that lbf-prelude::json class maps to
+pub trait Json {
+    fn to_json(&self) -> Result<Value, Error>;
+
+    fn from_json(value: Value) -> Result<Self, Error>
+    where
+        Self: Sized;
+}
```

Comment:
My reasoning for this was that Rust always uses the package name in imports, so I thought we don't need all the namespaces.
The crate name is `lbr-prelude`, so it would become `lbr-prelude::runtime::prelude::json::*`? Which already implies lambda buffers runtime prelude.
I generally agree with the notion of consistency, but I'm a bit indecisive here, if it's necessary here.

Reply:
I agree, let's keep it as it is, `lbr-prelude::Json` is sufficient!